model/group_status.go

```diff
@@ -13,6 +13,7 @@ import (
 type GroupStatus struct {
 	InstallationsTotal          int64
 	InstallationsUpdated        int64
+	InstallationsBeingUpdated   int64
```

Comment:
I like the idea of adding new field but I am not sure about the name, as by subtracting `updatedCount` and `awaitingUpdateCount` from `totalCount` we get all unstable Installations including those in failed states, hibernated or those being created/deleted.

I am afraid that is something that may prove a little confusing in the future. Maybe something like `InstallationsUnstable` would describe it more accurately?

Reply:
Yep, that's a good point. I will change the name here.